### PR TITLE
[AIRFLOW-XXX] Fix doc sphinx error

### PR DESCRIPTION
**Pull request by milton0825**
Make sure you have checked _all_ steps below.

### Jira

- [X] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [X] Here are some details about my PR, including screenshots of any UI changes:

Fix the Sphinx doc build error that is breaking master:
```
/home/travis/build/milton0825/airflow/docs/_api/airflow/contrib/operators/databricks_operator/index.rst:29: WARNING: Error in "function" directive:
invalid option data: extension option field name may not contain multiple words.
.. function:: _handle_databricks_operator_execution(operator, hook, log, context)
   Handles the Airflow + Databricks lifecycle logic for a Databricks operator
   :param operator: Databricks operator being handled
   :param context: Airflow context
/home/travis/build/milton0825/airflow/docs/_api/airflow/hooks/hive_hooks/index.rst:15: WARNING: Error in "function" directive:
unknown option: "return".
.. function:: get_context_from_env_var()
   Extract context from env variable, e.g. dag_id, task_id and execution_date,
   so that they can be used inside BashOperator and PythonOperator.
   :return: The context of interest.
looking for now-outdated files... none found
```

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [X] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [X] Passes `flake8`

### CODE REVIEW

**Comment by milton0825** [comment]
tests passed on: https://travis-ci.org/milton0825/airflow/builds/524299752

**Comment by XD-DENG** [comment]
Thanks @milton0825 for the fix.

I have restarted the failing test here. Will merge once it passes.

File: airflow/hooks/hive_hooks.py [hunk 1]
```diff
@@ -44,7 +44,8 @@ def get_context_from_env_var():
     """
     Extract context from env variable, e.g. dag_id, task_id and execution_date,
     so that they can be used inside BashOperator and PythonOperator.
-    :return: The context of interest.
+
+    :return The context of interest.
```
Comment (XD-DENG) [inline airflow/hooks/hive_hooks.py]: @milton0825 , I'm having a further check and may you share more details about this change? Not getting it.

**Comment by milton0825** [comment]
@XD-DENG tests passed!

File: airflow/hooks/hive_hooks.py (hunk 1 above)
Comment (milton0825) [reply airflow/hooks/hive_hooks.py]: I think we need an empty line between the description and the arg hint.

Comment (XD-DENG) [reply airflow/hooks/hive_hooks.py]: But you removed the`:` after `return`? Any reason for it?

Comment (milton0825) [reply airflow/hooks/hive_hooks.py]: I thought the right format is `:return` but looks like I was wrong after I did a search on Google.

**Pull request merged**